SU-SWS/stanford-jumpstart-deployer
# July 2017 Anchorage updates

# READY FOR REVIEW

# Summary
- Getting Anchorage up to speed with latest Drupal updates on Sites

# Needed By (Date)
- 7/25/17

# Criticality
- How critical is this PR on a 1-10 scale? 7

# Steps to Test

1. Just check that they build successfully. We've already tested on Anchorage. (The repo has changed structure, making adding Travis integration difficult.)

# Affected Projects or Products
- Anchorage sites

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/DEVOPS-86
- Anyone who should be notified? ( @annawatt @sherakama @kbrownell )
